src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+/**
+ *  A class which encapsulates the user interactions between Duke and the user.
+ *  @author  Chen Guanzhou
+ *  @version v1
+ */
+public class Ui {
+
+    /**
+     * A method for Duke to tell the user how many tasks they currently have.
+     */
+    public static String taskNumberMessage(TaskList taskList) {
+        if (taskList.getLength() == 1) {
+            return "Now you have 1 task in the list.";
+        }
+        else {
+            return "Now you have " + taskList.getLength() + " tasks in the list.";
+        }
+    }
+
+    /**
+     * Greeting message when Duke is first executed.
+     * @return
+     */
+    public static String greetingMessage() {
```

Comment:
Perhaps this method name can be getGreetingMessage()?

Reply:
I agree